doc/source/v2/user/index.rst

```diff
@@ -283,11 +283,11 @@ Replay Document Stream
 Bluesky is built around a streaming-friendly representation of data and
 metadata. (See event-model_.) To access the run---effectively replaying the
 chronological stream of documents that were emitted during data
-acquisition---use the ``canonical()`` method.
+acquisition---use the ``documents()`` method.
```

Comment:
Should we have a not about the rename here?

Reply:
Good suggestion. Added a note in `versionchanged` directive just below this in a new commit.